src/App.tsx

```diff
@@ -1,35 +1,51 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
-export class App extends React.Component<{}, State> {
-  state: State = {};
+export class App extends Component<{}, State> {
+  state: State = {
+    query: '',
+  };
+
+  onChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ query: event.target.value });
+  };
 
   render() {
+    const { query } = this.state;
+
+    const filteredMovies = moviesFromServer.filter(
+      movie => movie.title.toLowerCase()
+        .includes(query.toLowerCase()),
```

Comment:
you are only checking whether the title contains the query, but you need to also check the description

Reply:
Sorry, I thought that's optional in this task ("or" in README and in description of task)

Reply:
Yeah, it was optional. My bad